docs/tutorials/field_type/3_implement_the_tweet_type_class.md

```diff
@@ -27,12 +28,14 @@ public function getFieldTypeIdentifier()
 
 #### `createValueFromInput()` and `checkValueStructure()`
```

Comment:
maybe change title to `acceptValue()` and add as the first sentence:

`createValueFromInput()` and `checkValueStructure()` are used by the abstract Field Type implementation of `acceptValue()`.